app/src/main/res/layout/session_list_item.xml

```diff
@@ -57,7 +58,8 @@
         android:adjustViewBounds="true"
         android:baseline="13sp"
         android:contentDescription="@string/session_list_item_video_content_description"
-        android:src="@drawable/ic_change_video"
+        android:src="@drawable/ic_video"
+        android:tint="@color/session_list_item_video"
```

Comment:
Thanks for that referance, I actually had a really hard time understanding the different usages of the Icon, I'll use this as the base for the new version :+1:

Reply:
You are always welcome to ask me when you are stuck. Did you see the usage of `ic_without_video_recording_took_place` here?

https://github.com/EventFahrplan/EventFahrplan/blob/master/app/src/main/java/nerd/tuxmobil/fahrplan/congress/favorites/StarredListAdapter.kt#L74

Reply:
I must admit I am not too happy with the aesthetics of the new "blocky" camera icon. I looked around and found the following one:

- http://simpleicon.com/wp-content/uploads/video-camera-7.svg

Can you try how it looks with a red bar?

Reply:
This would be the Black Version with a Red Bar:
![video-camera-withBar](https://user-images.githubusercontent.com/44641688/194715037-e299b21e-bb6d-4369-a85a-9f3a6673fe23.svg)

Reply:
That looks really good. Do you like it, too?

Reply:
I'm not 100% sold on the Icon as I woul prefer a bit more simplistic Icon, but I wouldn't complain about it 👍

Reply:
Whoops, that should not have happend. I fixed that and double checked all Camera Icons, so they should now all be correct in color, size and having a bar or not